tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/server/package-info.java

```diff
@@ -0,0 +1,227 @@
+package org.eclipse.n4js.ide.tests.server;
+
+/**
+ * Documentation defining default projects or workspaces in
+ * {@link org.eclipse.n4js.ide.tests.server.AbstractStructuredIdeTest}s and for their creation in
+ * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceCreator}.
+ */
+class Documentation {
+	/**
+	 * If defined, a default npm project is used for all tests of one
```

Comment:
Instead of "If defined," maybe use the more detailed "If method `#getDefaultTestModules()` is overridden, ...". Makes this easier to understand (same at the beginning of the following documentation block).